samples/unit-tests/chart/getselectedseries/demo.html

```diff
@@ -0,0 +1,9 @@
+<script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
Are we actually using jQuery here? This also applies to many other of the new `demo.html` files. I think in the original visual samples we only used jQuery to add button click listeners.